frontend/components/events/EventPayment.tsx

```diff
@@ -1,27 +1,27 @@
 "use client";
 import { useState } from "react";
 
-import {
-  timestampToDateString,
-  timestampToTimeOfDay,
-} from "@/services/src/datetimeUtils";
-import {
-  CalendarDaysIcon,
-  ClockIcon,
-  CurrencyDollarIcon,
-  MapPinIcon,
-} from "@heroicons/react/24/outline";
+import { EventId } from "@/interfaces/EventTypes";
+import { timestampToDateString, timestampToTimeOfDay } from "@/services/src/datetimeUtils";
+import { getStripeCheckoutFromEventId } from "@/services/src/stripeService";
+import { CalendarDaysIcon, ClockIcon, CurrencyDollarIcon, MapPinIcon } from "@heroicons/react/24/outline";
 import { Timestamp } from "firebase/firestore";
 import Link from "next/link";
+import { useRouter } from "next/navigation";
 
 interface EventPaymentProps {
   date: Timestamp;
   location: string;
   price: number;
   vacancy: number;
+  stripe: boolean;
+  eventId: EventId;
+  isPrivate: boolean;
+  setLoading: (value: boolean) => void;
 }
 
 export default function EventPayment(props: EventPaymentProps) {
+  const router = useRouter();
   const [guestCount, setGuestCount] = useState(1);
```

Comment:
remove unused guestCount state variable

Reply:
i know this is bad practice as we should just remove code thats not needed, but since we are still in development phase, can i just leave it in here as its good stub code for implementing the feature for selecting more than 1 ticket. I have a jira ticket for that feature here already.
https://owenyang.atlassian.net/jira/software/c/projects/SPORTSHUB/boards/6?selectedIssue=SPORTSHUB-77

ill leave a comment in code, thanks